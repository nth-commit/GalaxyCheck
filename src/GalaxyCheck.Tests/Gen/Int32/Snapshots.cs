using Xunit;
using GC = GalaxyCheck;
using static Tests.TestUtils;

namespace Tests.Gen.Int32
{
    public class Snapshots
    {
        [Fact]
        public void Example() => SnapshotGenValues(GC.Gen.Int32());

        [Fact]
        public void ExampleOfExampleSpaces() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(-10, 10));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant1() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(0, 10).ShrinkTowards(5));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant2() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(-10, 0).ShrinkTowards(-5));
    }
}
