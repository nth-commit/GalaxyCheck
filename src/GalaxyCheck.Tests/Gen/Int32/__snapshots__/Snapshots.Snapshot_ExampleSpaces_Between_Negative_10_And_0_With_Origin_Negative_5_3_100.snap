-9
..-5
..-7
....-6
..-8
